frontend/lib/norent/footer.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { NorentRoutes as Routes } from "./routes";
+import { Link } from "react-router-dom";
+
+const MAILCHIMP_URL =
+  "https://nyc.us13.list-manage.com/subscribe/post?u=d4f5d1addd4357eb77c3f8a99&amp;id=588f6c6ef4";
+
+const EmailSignupForm = () => (
+  <form
+    action={MAILCHIMP_URL}
+    className="email-form is-horizontal-center"
+    method="post"
+    target="_blank"
+  >
+    <div className="mc-field-group">
+      <div className="control is-expanded">
+        <input
+          type="email"
+          name="EMAIL"
+          className="required email input"
+          id="mce-EMAIL"
+          placeholder="Enter your email"
+        />
+      </div>
+      <div className="control has-text-centered-touch">
+        <button className="button" type="submit">
+          ➔
```

Comment:
Huh for some reason I thought I already made this comment, but you might want to add an `aria-label` attribute here to provide alternative text for this button, since "right arrow" won't be very meaningful to screen reader users.

Reply:
Nice! Just tried this out with the VoiceOver screenreader and I can see the difference it makes.